docs/commands.md

```diff
@@ -10540,6 +10544,21 @@ hal plugins disable [parameters]
  * `--no-validate`: (*Default*: `false`) Skip validation.
 
 
+---
+## hal plugins disableDownloading
+
+Enable or disable jar downloading for all plugins
```

Comment:
Small nit, I feel like this should be a bit more descriptive. Who's downloading the plugins? Why do I need it? 

Also, I don't know how possible this is, but, since we _know_ we need this for environments running in Kubernetes should we automatically enable it if your `deploymentEnvironment` account is a Kubernetes accounts?

Reply:
I think the only way we can get these PRs merged is if we disable by default 😅